server/Dockerfile

```diff
@@ -1,132 +1,88 @@
+    RUN update-alternatives --set php /usr/bin/php7.4 && pecl channel-update pecl.php.net
     RUN cp "/usr/lib/$(gcc -dumpmachine)"/libfuzzy.* /usr/lib; pecl install ssdeep && pecl install rdkafka && pecl install simdjson
     RUN git clone --recursive --depth=1 https://github.com/kjdev/php-ext-brotli.git && \
         cd php-ext-brotli && phpize && ./configure && make && make install
 
-FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim" as python-build
+FROM "${DOCKER_HUB_PROXY}ubuntu:22.04" as python-build
     ENV DEBIAN_FRONTEND noninteractive
-    RUN apt-get update; apt-get install -y --no-install-recommends \
-        gcc \
-        git \
-        python3 \
-        python3-dev \
-        python3-pip \
-        python3-setuptools \
-        python3-venv \
-        python3-wheel \
-        libfuzzy-dev \
-        libffi-dev \
-        ca-certificates \
-        curl \
-        && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
-
+    RUN apt-get update; apt-get install -y --no-install-recommends python3-pip && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
     RUN mkdir /wheels
 
-    WORKDIR /tmp
-
-    RUN git clone --depth 1 https://github.com/CybOXProject/mixbox.git; \
-        cd mixbox || exit; python3 setup.py bdist_wheel -d /wheels; \
-        sed -i 's/-e //g' requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
-
-    # install python-maec
-    RUN git clone --depth 1 https://github.com/MAECProject/python-maec.git; \
-        cd python-maec || exit; python3 setup.py bdist_wheel -d /wheels
-
-    # install python-cybox
-    RUN git clone --depth 1 https://github.com/CybOXProject/python-cybox.git; \
-        cd python-cybox || exit; python3 setup.py bdist_wheel -d /wheels; \
-        sed -i 's/-e //g' requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
-
-    # install python stix
-    RUN git clone --depth 1 https://github.com/STIXProject/python-stix.git; \
-        cd python-stix || exit; python3 setup.py bdist_wheel -d /wheels; \
-        sed -i 's/-e //g' requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
-
-    # install STIX2.0 library to support STIX 2.0 export:
-    # Original Requirements has a bunch of non-required pacakges, force it to only grab wheels for deps from setup.py
-    RUN git clone --depth 1 https://github.com/MISP/cti-python-stix2.git; \
-        cd cti-python-stix2 || exit; python3 setup.py bdist_wheel -d /wheels; \
-        echo "-e ." > requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
+    # Grab module wheels we need
+    RUN pip3 wheel --no-cache-dir -w /wheels/ plyara pyzmq redis python-magic lief>=0.13.1 pymisp pydeep2 misp-lib-stix2 stix cybox maec mixbox
 
-    # Install the new build tool
-    RUN pip3 install build
+FROM "${DOCKER_HUB_PROXY}ubuntu:22.04"
 
-    # install PyMISP
```

Comment:
We get these wheels in the python-builder stage

Reply:
ACK